TDTChocolate/FoundationAdditions/NSDictionary+TDTHTTPAdditions.h

```diff
@@ -0,0 +1,19 @@
+//
+//  NSDictionary+TDTHTTPAdditions.h
+//  Talkto
+//
+//  Created by Amandeep Singh on 8/24/17.
+//  Copyright © 2017 Talk.to FZC. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@interface NSDictionary (TDTHTTPAdditions)
+
+/**
+ This method generates a HTTP Request query string 
+ from the dictionary's key value pairs.
+ */
+- (NSString *)tdt_queryString;
```

Comment:
`TDTChocolate` already has a similar function [`tdt_stringFromQueryComponents`](https://github.com/talk-to/Chocolate/blob/master/TDTChocolate/FoundationAdditions/TDTURLQueryComponents.h#L59). Does this do something different?